@utahdts/utah-design-system/@types/jsDocTypes.d.js

```diff
@@ -233,6 +233,18 @@
  * }
  */
 
+// ========== File Input ========== //
+/**
+ * @typedef {[FileContextValue]} FileContext
```

Comment:
If the context is a single value, then probably don't have to put it in an array. If it's to allow future expansion then it may be better in an object. If it's to mimic a state value/setter it should have a setState ([FileContextValue, import('use-immer').Updater<FileContextValue>]).

Reply:
Yes.